ush/forecast_postdet.sh

```diff
@@ -497,229 +469,198 @@ EOF
       logo=${COM_ATMOS_HISTORY}/${RUN}.t${cyc}z.atm.logf${FH3}.txt
       pgbo=${COM_ATMOS_MASTER}/${RUN}.t${cyc}z.master.grb2f${FH3}
       flxo=${COM_ATMOS_MASTER}/${RUN}.t${cyc}z.sfluxgrbf${FH3}.grib2
-      eval $NLN $atmo $atmi
-      eval $NLN $sfco $sfci
-      eval $NLN $logo $logi
-      if [ $WRITE_DOPOST = ".true." ]; then
-        eval $NLN $pgbo $pgbi
-        eval $NLN $flxo $flxi
+      eval "${NLN}" "${atmo}" "${atmi}"
+      eval "${NLN}" "${sfco}" "${sfci}"
+      eval "${NLN}" "${logo}" "${logi}"
+      if [[ ${WRITE_DOPOST} = ".true." ]]; then
+        eval "${NLN}" "${pgbo}" "${pgbi}"
+        eval "${NLN}" "${flxo}" "${flxi}"
```

Comment:
Let's get rid of these `eval`s. Also `${NLN}` is often multiple words that we need to keep separate (i.e. `ln -s`).
```suggestion
      ${NLN} "${atmo}" "${atmi}"
      ${NLN} "${sfco}" "${sfci}"
      ${NLN} "${logo}" "${logi}"
      if [[ ${WRITE_DOPOST} = ".true." ]]; then
        ${NLN} "${pgbo}" "${pgbi}"
        ${NLN} "${flxo}" "${flxi}"
```